heat/core/statistics.py

```diff
@@ -498,13 +529,15 @@ def local_max(*args, **kwargs):
             result = result[0]
         return result
 
-    return operations.__reduce_op(x, local_max, MPI.MAX, axis=axis, out=out, keepdim=keepdim)
+    return operations.__reduce_op(
+        x, local_max, MPI.MAX, axis=axis, out=out, keepdim=keepdim
+    )
 
 
 def maximum(x1, x2, out=None):
     """
-    Compares two tensors and returns a new tensor containing the element-wise maxima. 
-    If one of the elements being compared is a NaN, then that element is returned. TODO: Check this: If both elements are NaNs then the first is returned. 
+    Compares two tensors and returns a new tensor containing the element-wise maxima.
+    If one of the elements being compared is a NaN, then that element is returned. TODO: Check this: If both elements are NaNs then the first is returned.
```

Comment:
Seems like an overlong line. Shouldnt this be auto-moved to the next line?

Reply:
In terms of line length, Black uncompromisingly does not wrap at least the doc strings. Other comments are wrapped, AFAIR. Here, it only removed trailing white space.